leap.py

```diff
@@ -1,14 +1,13 @@
 # Function definition is here
 def is_leap_year( year ):
-   if year%4 == 0 and year%100 != 0:
-       print True
+   if (year%4 == 0 and year%100 != 0) or year%400 ==0:
```

Comment:
I guess that's just a typo - a missing space after `==` in `year%400 ==0`.

Besides that grouping and spacing look nicely readable 👍See https://www.python.org/dev/peps/pep-0008/#other-recommendations ("If operators with different priorities are used....").

Reply:
Regarding indentations - it's common to add 4 spaces for each new indentation block. See https://www.python.org/dev/peps/pep-0008/#indentation.